submissions/src/main/resources/application.properties

```diff
@@ -4,4 +4,15 @@ azure.storage.connection-string=${AZURE_STORAGE_CONNECTION_STRING:DefaultEndpoin
 azure.storage.container-name=${AZURE_STORAGE_CONTAINER_NAME:reports}
 azure.storage.queue-name=${AZURE_STORAGE_QUEUE_NAME:elr-fhir-receive}
 azure.storage.table-name=${AZURE_STORAGE_TABLE_NAME:submission}
-spring.security.oauth2.resourceserver.jwt.issuer-uri=https://reportstream.oktapreview.com/oauth2/ausekaai7gUuUtHda1d7
\ No newline at end of file
+spring.security.oauth2.resourceserver.jwt.issuer-uri=https://reportstream.oktapreview.com/oauth2/ausekaai7gUuUtHda1d7
+
+# Allowed headers
+allowed.headers.client_id=client_id
```

Comment:
Few things:

This being a map doesn't seem to serve a purpose since both key/value are the same(ish) and you ignore the key in code anyway. In code the type will just be `List<String>`.

I think adding an additional prefix would help clarify the configuration as well. Something like `event.allowed.headers`?

Last bit is I think yml really simplifies this file over a properties file. Here's what the result would look like:
```yml
event.allowed:
  headers:
    - client_id
    - content-type
    - payloadname
    - x-azure-clientip
    - content-length
  queryParameters:
    - ???
```

With a properties file there is much more duplication and you need to add indices.
```properties
event.allowed.headers[0]=client_id
event.allowed.headers[1]=content-type
...
```

Reply:
got it fixed like this. I did leave it as just `allowed` since this list is also used to filter out parameters for the queue message